src/Azure.Functions.Cli/StaticResources/local.settings.json.template

```diff
@@ -2,6 +2,7 @@
   "IsEncrypted": false,
   "Values": {
     "FUNCTIONS_WORKER_RUNTIME": "{FUNCTIONS_WORKER_RUNTIME}",
-    "AzureWebJobsStorage": "{AzureWebJobsStorage}"
+    "AzureWebJobsStorage": "{AzureWebJobsStorage}",
+    "AZURE_FUNCTIONS_ENVIRONMENT": "Development"
```

Comment:
However, this will only help for new apps. Maybe we should also force dev mode imperatively when the cli builds the host?

Reply:
I'm wondering if it'll be better to do that in the cli code before we invoke the webhost. That way you'll always get it if it's not defined. The problem is that during publishing, this file gets turned into App Settings. I'd think most people won't want that by default when they publish. We can exclude it, but then will need to add a flag is someone actually wants it. What do you think?

Reply:
Ah, based on the file name I assumed this was ONLY for running locally. Yes, we don't want this pushed. I'll set this imperatively instead.

Reply:
I thought the settings in local.settings.json are ONLY for running locally. That's what our documentation says here: https://docs.microsoft.com/en-us/azure/azure-functions/functions-run-local. 

> "Settings in the local.settings.json file are only used by Functions tools when running locally. By default, these settings are not migrated automatically when the project is published to Azure."

Reply:
the file is only loaded locally. That's used when people are working locally with the cli and need to set various App Settings etc. on publish you can pass a `--publish-local-settings` which will parse this file and update the Azure App Settings with the values from here.